sig-TEMPLATE/sig-charter-template.md

```diff
@@ -0,0 +1,32 @@
+# SIG YOURSIG Charter
+
+## Scope
+
+Include a 2-3 sentence summary of what work SIG TODO does. Imagine trying to
```

Comment:
This template should be empty aside form the headings right?

Reply:
I'd liek to keep the description which should then be replaced.